packages/react-scripts/scripts/utils/frontierInit.js

```diff
@@ -19,17 +19,17 @@ function installFrontierDependencies(appPath, appName, ownPath) {
   depsToInstall.push(
     ...[
       '@emotion/core@10',
-      '@fs/zion-axios@1',
-      '@fs/zion-cache@1',
-      '@fs/zion-frontend-friends@1',
-      '@fs/zion-icon@2',
-      '@fs/zion-locale@2',
-      '@fs/zion-root@2',
-      '@fs/zion-router@2',
-      '@fs/zion-style-normalize@1',
-      '@fs/zion-subnav@1',
-      '@fs/zion-user@2',
-      '@fs/zion-ui@5',
+      '@fs/zion-axios@2.0.0-alpha.1',
+      '@fs/zion-cache@2.0.0-alpha.1',
+      '@fs/zion-frontend-friends@2.0.0-alpha.2',
+      '@fs/zion-icon@3.0.0-alpha.1',
+      '@fs/zion-locale@3.0.0-alpha.2',
+      '@fs/zion-root@3.0.0-alpha.3',
+      '@fs/zion-router@3.0.0-alpha.1',
+      '@fs/zion-style-normalize@3.0.0-alpha.2',
+      '@fs/zion-subnav@2.0.0-alpha.1',
+      '@fs/zion-user@3.0.0-alpha.1',
+      '@fs/zion-ui@6.0.0-alpha.2',
```

Comment:
It looks like some changes that are in latest 5 did not make it into this alpha version of 6 (specifically the fix of the scollbars on tabs)

Reply:
Good eye. I'll make sure when I publish zion-ui@6, that is has those fixes.